test/test.customdriver.js

```diff
@@ -43,7 +43,7 @@ describe('When Custom Drivers are used', function() {
         });
     });
 
-    it('fails to define a custom driver with overlapping driver name',
+    xit('fails to define a custom driver with overlapping driver name',
```

Comment:
Should these still be marked as pending? That's what `xit` does, right? Should we remove them or should they be passing?

Reply:
I think that we should remove these completely.
We always allowed redefining custom drivers but prohibited redefining internal drivers, possibly without any specific reason in mind.
Allowing to override the internal drivers as well can let us bypass issues like #676.
If you are fine with this, I'm going to rebase, delete the `xit`ed tests and merge this today and you can release this as the next patch release for v1.5 whenever works best for you.

Reply:
Ah, that makes sense. Yeah, I think we did it originally to make sure users didn’t break anything. But I think allowing them to refine them is a good idea, like you say.

Maybe we could log a console message if they redefine any driver though? That way they at least know what’s up. But yeah, removing these and merging sounds good! 👍🏻

Reply:
I remember a case that a user complained about some logs on perfectly valid use cases, but we can do that here and check their reactions.
I'm going to use `console.info` if that's ok with you (hopefully later today).

Reply:
Ah, right. I think it's worth trying... users can always build a version that omits console statements... or maybe we could build our own logger in the future that can be disabled.

Anyway yeah info or even debug is fine for this 👍